sdk/src/main/java/io/opentelemetry/sdk/metrics/DoubleMeasureSdk.java

```diff
@@ -46,7 +48,12 @@ public void record(double value, LabelSet labelSet) {
 
```

Comment:
I know it is a WIP, but should the constructor initialize an Aggregation?

Reply:
The goal for the moment is to get Counters working :)